charts/datadog/templates/_container-agent.yaml

```diff
@@ -242,6 +242,9 @@
     {{- if .Values.datadog.kubelet.hostCAPath }}
 {{ include "datadog.kubelet.volumeMount" . | indent 4 }}
     {{- end }}
+{{- if eq (include "should-mount-fips-configmap" .) "true" }}
+{{- include "linux-container-fips-proxy-cfg-volumemount" . | nindent 4 }}
+{{- end }}
```

Comment:
it should be in the `{{- if eq .Values.targetSystem "linux" }}` condition ([line 161](https://github.com/DataDog/helm-charts/pull/1095/files#diff-0973c8baa0f902471c709f7be2b0682aa0b89d92368289528c1d4b42118980a3L161))